### ISSUE TITLE
List view for all users

### ISSUE THREAD

**Issue by henrietta-k** [uchicago-cs/chigame#112]

**Comment by AndrewM131** [comment]
Description: We want to display a view that displays all users and can be accessed by all users. There will be limited information, and no private information should be displayed. Pull request related is #120

**Comment by AndrewM131** [comment]
Update: We met November 2nd to go over this issue, and after talking about it, and coding on Henrietta's laptop, she made the pull request. It is currently under review, and needs some changes before asking for another review.

**Comment by AndrewM131** [comment]
Update: We addressed the changes from the first review, and are awaiting the review from our TA.

**Comment by AndrewM131** [comment]
Update: After reevaluating how to implement the UserTable after the two reviews, we successfully implemented it, and got the PR approved.

**Comment by AndrewM131** [comment]
Closing statement: We successfully implemented a list view for all users by implementing a UserTable. Any user can access this view.
In the process of the task, we met two times, the first was on November 8th when we worked together to figure out the specific goals that this issue would address (setting up a page so that in the future, tournament information for all users on the platform can be displayed here) and how we wanted to go about accomplishing this goal. We met again on November 12th to finalize some issues (linting issues) and fine-tune what our goals were related to this task. During our meetings, Henrietta coded on her laptop while Ishi and I assisted by using our laptops to help research and look at chigame code. I also assisted in the github issue updates, and Henrietta took care of resolving the review comments.
